app/controllers/delivery_options_controller.rb

```diff
@@ -0,0 +1,62 @@
+class DeliveryOptionsController < ApplicationController
+  before_action :set_seller
+  before_action :set_seller_delivery_options, only: [:update, :destroy]
+
+  # GET /sellers/:seller_id/delivery_options
+  def index
+    delivery_options = @seller.delivery_options.map do |delivery_option|
+      delivery_type = delivery_option.delivery_type
+      json = delivery_option.as_json
+      json['delivery_type'] = delivery_type.as_json
+      json
+    end
+
+    json_response(delivery_options.sort_by{|de| de['delivery_type']['name'].to_i})
```

Comment:
There is a way to do this in the query, but maybe you can do that as a follow up to unblock Olivia? Just create a separate Trello task. But here we are doing a N log N sort for every Seller. Not terrible but worth filing

Reply:
https://trello.com/c/21DtS7WI